docs/content/preview/yugabyte-voyager/migrate/assess-migration.md

```diff
@@ -16,31 +17,42 @@ The Voyager Migration Assessment feature is specifically designed to optimize th
 
 ## Overview
 
-Voyager collects various metadata or metrics from the source database, such as table columns metadata, sizes of tables/indexes, read/write IOPS for tables/indexes, and so on. With this data, an assessment report is prepared which covers the following key details:
+When you run an assessment, Voyager collects metadata or metrics from the source database. This includes table columns metadata, sizes of tables and indexes, read and write IOPS for tables and indexes, and so on. With this data, Voyager generates an assessment report with the following key details:
 
-- **Database compatibility**: Assesses the compatibility of the source database with YugabyteDB, identifying unsupported features and data types.
+- **Database compatibility**. An assessment the compatibility of the source database with YugabyteDB, identifying unsupported features and data types.
```

Comment:
Why don't these categories match the headings/items in the actual report (shown below)

Reply:
cc @sanyamsinghal @makalaaneesh